internal/system/agent/router.go

```diff
@@ -113,14 +129,14 @@ func configHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	w.Header().Add("Content-Type", "application/json")
+	w.Header().Add(clients.ContentType, clients.ContentTypeJSON)
```

Comment:
- This update, involving `clients.ContentType` etc. here (and similar updates elsewhere in this PR) is fine by me. I have recognized, though, that the _other_ style, involving strings instead, such as `"Content-Type"` etc., is heavily prevalent in the code base.
- While this may sounds like a nit, having a consistent style across the code base promotes easier comprehension by making searches (`grepping`) that much easier 👍

Reply:
We should utilize the constants here IMHO.